plugins/scaffolder/src/next/TemplateListPage/TemplateListPage.tsx

```diff
@@ -153,22 +157,20 @@ export const TemplateListPage = (props: TemplateListPageProps) => {
     <EntityListProvider>
       <Page themeId="website">
         <Header
-          pageTitleOverride="Create a new component"
-          title="Create a new component"
-          subtitle="Create new software components using standard templates in your organization"
+          pageTitleOverride={t('create_a_new_component')}
+          title={t('create_a_new_component')}
+          subtitle={t('template_list_page_subtitle')}
```

Comment:
Here seems like a good example of which approach would be better, as it contains a mix of styles.
The first style is to name a key as a current label of the header, another style is the positioning of the subtitle.

I agree with [your suggestion and staying flat in key naming ](https://github.com/backstage/backstage/pull/20130). 
And question renames which style to use.

This i18n capabilities is a good approach not only to translate the labels 1:1 as they are right now, but also override them. If company X want to have instead of "Create a new component", having "Self Service Templates" it would be hard to maintain the file which looks like 

`create_a_new_component = 'Self Service Templates'`

Can be same for the buttons, labels, etc. 

By creating a key with a name 'create' and reusing in a button, label, header, etc. on different pages at some point could be not scalable, as now you want to have a name in the button 'Run', and the label on other page with the same key 'create' has a different context, but will be affected as well.